config/config.go

```diff
@@ -41,6 +42,7 @@ func Get() (*Config, error) {
 		Brokers:                    []string{"localhost:9092"},
 		KafkaMaxBytes:              2000000,
 		ImageUploadedTopic:         "image-uploaded",
+		StaticFilePublishedTopic:   "static-file-published",
```

Comment:
More a general question for the whole PR rather than this line specifically. Is the queue for 'published' (past tense) or for 'to-be-published' images. Ie. the static file publisher will be picking things off this queue and doing the publishing. At the point they are put in this queue they are not "published".

Reply:
I think you are right that the naming of the topic is not very good, as this queue will be consumed by the service responsible for the actual publishing. Shall I change it to `static-file-to-be-published`? (I will need to change it here and also in dp-config)

Reply:
Not precious about any naming, but I wanted to add some clarity to the process and the exact time something is `published`. By the time this app is decrypting the files the resource is actually already marked as `published` on the API and we will be decrypting and streaming the files on the fly immediately at publish time. When the images are made publicly available we will then be permanently decrypting them using this app.

Reply:
After the discussion, I will leave the topic name as `static-file-published`. In the next task (implement publish endpoint) I will change the state machine as discussed.